# Completá cada bloque añadiendo los except específicos necesarios
# (no usar except hasta atrapar todos los errores presentes en el ejercicio)


try:
    datos = ["5", "0"]
    numero = int(datos[0])
    divisor = int(datos[1])
    resultado = numero / divisor
    print("Programa correcto")

#Completar aquí